fe/src/app/jobs/[jobId]/offer/page.tsx

```diff
@@ -0,0 +1,119 @@
+"use client"
+import { Breadcrumbs, Button, Input, Textarea, Typography } from "@/lib/materialTailwindExports"
+import { OfferType } from "@/types/OfferType"
+import { getToken } from "@/utils/getToken";
+import { BASE_API_URL, SERVICE_JOBS_ROUTE } from "@/utils/urls";
+import { useParams, useRouter } from 'next/navigation';
+import { useState } from "react"
+import toast from "react-hot-toast";
+import Datepicker, { DateValueType } from "react-tailwindcss-datepicker";
+import { date, number, object, string } from "yup"
+
+const offerFormSchema = object({
```

Comment:
Maybe extract yup schema and default values into utils.ts just like in /advert -> utils.ts?